dashboard/src/utils/dateFunctions.js

```diff
@@ -30,3 +30,28 @@ export const getTodayMidnightUTCDate = () => {
   midnightUTC.setUTCHours(0, 0, 0, 0);
   return midnightUTC;
 };
+
+/**
+ * Convert a date string into Locale Date String with seconds and milli seconds removed *
+ * @function
+ * @param {string} strDate - Date in string format
+ * @return {string} - Locale Date string
+ */
+
+export const formatDateTime = (dateTimeStamp) => {
+  const dateObj = new Date(dateTimeStamp);
+  dateObj.setSeconds(0, 0);
+  return dateObj.toLocaleString();
+};
+/**
+ * Find number of days between given date and today's date*
+ * @function
+ * @param {string} dateString - Date in string format
+ * @return {number} - Number of days in number format
+ */
+export const findNoOfDays = (dateString) => {
+  const deletionDate = new Date(dateString);
+  const today = new Date();
+  const diffTime = Math.abs(deletionDate - today);
+  return Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+};
```

Comment:
While this is a nicely generic function, given the way that it is used, if `dateStr` is now in the past, then it would probably be better if it returned zero (or even a negative number), because returning a positive number in that case is ambiguous.

Reply:
Got it!